manifests/init.pp

```diff
@@ -170,6 +170,10 @@
   include dns::config
   contain dns::service
 
+  if $group_manage {
+    group { $group: }
+  }
```

Comment:
Normally the group is created by the package in install.pp. Perhaps it should be there and also chained to first install the package and then manage the group.

Reply:
Moved to `install.pp` with dependencies.